.idea/codeStyles/Project.xml

```diff
@@ -32,7 +32,16 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
Do we really want to include these `Project.xml` changes here or were they included by mistake?

Reply:
ah no, they are here by mistake. They keep on appearing in my changes and I always have to manually remove them :/